### PR TITLE
Basic skeleton for create-sibling-dataverse

### PR DESCRIPTION
**Pull request by bpoldrack**
This is a skeleton for the `create-sibling-dataverse` command with all the boilerplate of a command implementation.
It's supposed to be the basis, so we can work on different parts of it in parallel.
The interface and structure are taken from `create-sibling-webdav` of the `datalad-next` extension, since the underlying mode of operation is likely similar. However, nothing in there is set in stone.

### CODE REVIEW

**Comment by mih** [comment]
Great, thx! We need to make the test conditional on dataverse actually being deployed -- not the case for all CI runs.

**Comment by bpoldrack** [comment]
@mih True.

Suggestion (should be added to README if agreed, I think): https://github.com/datalad/datalad-dataverse/pull/35/commits/f82f29d7c649409815a0f440640161f43851ea67

File: datalad_dataverse/create_sibling_dataverse.py [hunk 1]
```diff
@@ -0,0 +1,264 @@
+"""High-level interface for creating a combi-target on a WebDAV capable server
```
Comment (mih) [inline datalad_dataverse/create_sibling_dataverse.py]: Docs need update. C&P leftover.

File: datalad_dataverse/tests/test_create_sibling_dataverse.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+from datalad_dataverse.tests import (
+    API_TOKENS,
+    DATAVERSE_URL
+)
+
+
+@skip_if(cond=not DATAVERSE_URL)
+@with_tempfile
+def test_dummy(path=None):
```
Comment (mih) [inline datalad_dataverse/tests/test_create_sibling_dataverse.py:24]: I'd say we skip the test until there is an actual implementation, and then merge this PR. OK @bpoldrack ?

**Review by mih — CHANGES_REQUESTED** [review]

File: datalad_dataverse/tests/test_create_sibling_dataverse.py (hunk 2 above)
Comment (bpoldrack) [reply datalad_dataverse/tests/test_create_sibling_dataverse.py:24]: Ok. Just wanted to show how to use that decorator

**Comment by mih** [comment]
Thx!

**Pull request merged**